balloc/balloc.c

```diff
@@ -3012,26 +3044,29 @@ M0_INTERNAL void m0_balloc_init(struct m0_balloc *cb)
 	cb->cb_ballroom.ab_ops = &balloc_ops;
 }
 
-M0_INTERNAL int m0_balloc_create(uint64_t              cid,
-				 struct m0_be_seg     *seg,
-				 struct m0_sm_group   *grp,
-				 struct m0_balloc    **out,
-				 const struct m0_fid  *fid)
+M0_INTERNAL int m0_balloc_create(uint64_t			 cid,
+				 struct m0_be_seg		*seg,
+				 struct m0_sm_group		*grp,
+				 struct m0_ad_balloc_format_req	*bcfg,
+				 struct m0_balloc		**out)
```

Comment:
Usually, we put "TAB" at the beginning of a line, not in between chars.
Here, please remove the TABs in type and variables.

Reply:
Taken care..